### DISCUSSION TITLE
Important Note Regarding Pull Requests

### DISCUSSION THREAD

**Discussion by coopsand** [USU-ITLS-x265-23-summer/class-repo#97] [General]
Hello everyone!

TL;DR: For the fastest possible updates to the repo - don't request specific people as reviewers.

I have noticed over the past couple of weeks that some pull-requests have requested members on them. My understanding is while this is a good practice for some pull requests, it makes it so that **only the requested people can accept and merge the pull request**. I have wanted to approve some posts but haven't been able to due to me not being a requested reviewer.

If you don't request any reviewers, **anyone** will be able to accept and merge your pull request - making it so the changes or additions you want to make to the repo get approved quicker.

**Comment by jakegros** [comment]
I have run into the same issue, thus have found it difficult to participate at all in approving.
Here is some documentation on it:
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#about-pull-requests
